docker/scripts/checkCustomFiles.sh

```diff
@@ -0,0 +1,6 @@
+if [[ -e /custom_files/ ]]; then
+	source /root/.bash_profile; \
+	sleep 5; \
+	/bin/cp --force /custom_files/cbiologo.png $CATALINA_HOME/webapps/cbioportal/images;
```

Comment:
Would it be possible to customise these settings using the [cBioSkins](https://github.com/cBioPortal/cbioportal/wiki/Customizing-your-instance-of-cBioPortal) configuration options? Then there would be no need to run an additional shell script concurrently with Catalina.